source/type/pb_converter.cpp

```diff
@@ -71,8 +71,19 @@ struct PbCreator::Filler::PtObjVisitor: public boost::static_visitor<> {
         add_pt_object(bo);
     }
     void operator()(const nt::disruption::LineSection& line_section) const {
-        //TODO: for the moment a line section is only a line, but later we might want to output more stuff
-        add_pt_object(line_section.line);
+        for(const nt::Route* route: line_section.routes) {
+            auto* pobj = add_pt_object(route);
```

Comment:
we want to output line sections as routes with stop impacted stops ?

I'm quite OK with this, but @stifoon what do you think about this ?

Isn't it a pity to loose the `line section` information ?